lxd/instance_post.go

```diff
@@ -491,6 +492,14 @@ func instancePostMigration(s *state.State, inst instance.Instance, newName strin
 			instProfiles = append(instProfiles, p.Name)
 		}
 
+		// Get current root disk device from expanded devices.
+		rootDevKey, rootDev, err = shared.GetRootDiskDevice(inst.ExpandedDevices().CloneNative())
+		if err != nil {
+			return err
+		}
+
+		var profRootDev map[string]string
```

Comment:
What is "prof" here?

Reply:
profile :sweat_smile:

Reply:
ah can just use "profileRootDev" its not too long.